qemu/tests/virtio_serial_file_transfer.py

```diff
@@ -161,7 +161,35 @@ def transfer_data(params, vm, host_file_name=None, guest_file_name=None,
     :param sender: Who send the data file
     :return: True if pass, False and error message if check fail
     """
+    def kill_pid():
+        """
+        Kill serial script process on host and guest to avoid two threads run
+        simultaneously.
+        """
+        target_process = params["target_process"]
+        list_cmd = params["list_cmd"]
+        kill_cmd = params["kill_cmd"]
+
+        host_process_cmd = 'ps aux | grep "serial_host_send_receive.py"'
```

Comment:
better to use `pgrep`.

Reply:
There is multi python pid, can not get the exact pid I want via pgrep command.